documentation/whatsnew/fusetools/fusetools-news-10.3.0.AM1.adoc

```diff
@@ -0,0 +1,26 @@
+= Fuse Tooling - What's New in 10.3
+:page-layout: whatsnew
+:page-component_id: fusetools
+:page-component_version: 10.3.0.AM1
+:page-product_id: jbt_core
+:page-product_version: 4.5.3.AM1
+
+== New shortcuts in Fuse perspective
+
+Shortcuts for Java, Launch and Debug configuration and basic navigation are now enabled by default with Fuse perspective.
+
+The result is a set of actions now available in the Toolbar:
+
+image::./images/newToolbarAction.png[New Toolbar action]
+
+All the associated key shortcuts are also available, such as Ctrl+Shift+T to open a Java Type.
+
+== Performance improvement: loading Advanced tab for Camel components
+
+The load of Advanced tab in Properties view for Camel components has been greatly improved.
+
+image::./images/advancedTab.png[Advanced Tab in Properties view]
+
+In case of Camel Components having a lot of parameters, it was taking several seconds. For instance with the File component, it was taking 3.5s. It is now taking 350ms, response time has been divided by 10. Not ideal but enough to not break anymore user's flow of thought. (see this https://www.nngroup.com/articles/response-times-3-important-limits/[old but interesting article on response time])
+
+In case you notice some places where you feel that the performance really needs to be improved, please report it (or vote if already reported) on the https://issues.jboss.org/browse/FUSETOOLS[Fuse Tooling tracker]. It will help driving the development priorities to improve your user experience.
```

Comment:
"If you notice some other places lacking performance please report it using the https://issues.jboss.org/browse/FUSETOOLS[Fuse Tooling issue tracker]. We really appriciate your feedback and it will also help to improve the user experience."